### PR TITLE
[.NET] Add Brazilian Portuguese (pt-BR) locale for .Net BotBuilder

### PR DESCRIPTION
**Pull request by brunoalbano**
* Added Brazilian Portuguese locale files for .Net BotBuilder.

### CODE REVIEW

File: CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+          <source>Quit</source>
+          <target state="signed-off">Sair</target>
+        </trans-unit>
+        <trans-unit id="CommandQuitHelp" translate="yes" xml:space="preserve">
+          <source>Quit: Quit the form without completing it.</source>
+          <target state="signed-off">Sair: Sair do formulário sem completá-lo.</target>
+        </trans-unit>
+        <trans-unit id="CommandQuitTerms" translate="yes" xml:space="preserve">
+          <source>quit;stop;finish;goodby?;good bye?;bye;ciao;adios;bye-bye;so long;cheers</source>
+          <target state="signed-off">sair;cancelar;parar;terminar;finalizar;encerrar;bye;tchau?;adeus;flw;t+;até mais</target>
```
Comment (msft-shahins) [inline CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf]: >flw [](start = 102, length = 3)

What is this abbreviating in Portuguese>

File: CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+        </trans-unit>
+        <trans-unit id="CommandReset" translate="yes" xml:space="preserve">
+          <source>Start over</source>
+          <target state="signed-off">Recomeçar</target>
+        </trans-unit>
+        <trans-unit id="CommandResetHelp" translate="yes" xml:space="preserve">
+          <source>Reset: Start over filling in the form.  (With defaults from your previous entries.)</source>
+          <target state="signed-off">Recomeçar: Recomeçar o preenchimento do formulário. (Com valores predeterminados com base no preenchimento anterior.)</target>
```
Comment (msft-shahins) [inline CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf]: >predeterminados com base [](start = 102, length = 24)

"baseados no"

File: CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+          <source>start over;reset;clear</source>
+          <target state="signed-off">reiniciar;recomeçar;resetar;limpar;iniciar novamente;começar novamente</target>
+        </trans-unit>
+        <trans-unit id="CommandStatus" translate="yes" xml:space="preserve">
+          <source>Status</source>
+          <target state="signed-off">Situação</target>
+        </trans-unit>
+        <trans-unit id="CommandStatusHelp" translate="yes" xml:space="preserve">
+          <source>Status: Show your progress in filling in the form so far.</source>
+          <target state="signed-off">Situação: Exibe o progresso de preenchimento do formulário até o momento.</target>
```
Comment (msft-shahins) [inline CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf]: >de [](start = 65, length = 2)

no

File: CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+          <source>Please select one or more {&amp;} {||}</source>
+          <target state="signed-off">Por favor, selecione um(a) ou mais {&amp;} {||}</target>
+        </trans-unit>
+        <trans-unit id="TemplateEnumSelectOne" translate="yes" xml:space="preserve">
+          <source>Please select a {&amp;} {||}</source>
+          <target state="signed-off">Por favor, selecione um(a) {&amp;} {||}</target>
+        </trans-unit>
+        <trans-unit id="TemplateFeedback" translate="yes" xml:space="preserve">
+          <source>For {&amp;} I understood {}. {?"{0}" is not an option.}</source>
+          <target state="signed-off">Com {&amp;} eu entendi {}. {?"{0}" não é uma opção.}</target>
```
Comment (msft-shahins) [inline CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf]: >  [](start = 59, length = 1)

que

**Comment by msft-shahins** [comment]
@brunoalbano thanks for your translation. Feel free to address the comments that my colleague made about the translations.

File: CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf (hunk 4 above)
Comment (brunoalbano) [inline CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf]: I didn't understand, was it supposed to be _"Com {&amp;} eu entendi **que** {}. {?"{0}" não é uma opção.}'_ ? It sounds strange in this way.

File: CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf (hunk 1 above)
Comment (brunoalbano) [inline CSharp/Library/Microsoft.Bot.Builder/MultilingualResources/Microsoft.Bot.Builder.pt-BR.xlf]: _flw_ is the abbreviation to _falou_, a informal way to end a conversation

**Pull request merged**